tfjs-converter/python/tensorflowjs/converters/keras_tfjs_loader.py

```diff
@@ -108,6 +170,30 @@ def _get_weights_manifest_from_config_json(config_json):
         'Field "weightsManifest" is missing from the JSON content.')
   return config_json['weightsManifest']
 
+def _generate_v3_keys(config):
+  if isinstance(config, dict):
+    list_of_keys = list(config.keys())
+    for key in list_of_keys:
+      _generate_v3_keys(config[key])
+    if 'class_name' in list_of_keys:
+      config['module'] = _get_module(config['class_name'])
+      # Put registred name as None since we do not support
+      # custom object saving when we save the model.
+      config['registered_name'] = None
```

Comment:
The structure of this function looks similar to `_discard_v3_keys`. They're both iterating recursive list / dicts. However, I don't see a good way to extract that iteration logic out of them since this function needs to check what keys the map has. No action necessary.